services/add-component-handler.test.js

```diff
@@ -0,0 +1,44 @@
+var dirname = __dirname.split('/').pop(),
+  filename = __filename.split('/').pop().split('.').shift(),
+  lib = require('./add-component-handler'),
+  dom = require('@nymag/dom');
+
+describe(dirname, function () {
+  describe(filename, function () {
```

Comment:
test the logic of the default exported function (`addComponentHandler()`) as well